scripts/update-version.sh

```diff
@@ -1,12 +1,16 @@
+#! /bin/sh
+
+: "${FD:=fd}"
```

Comment:
Do we really need this? (can we add a comment on how to install fd (via cargo and apt-get)

Reply:
On MacOS, `fd` can also be installed using homebrew.

Reply:
I added this because when you install `fd` with apt-get, the command is `fdfind`. I assume we need to support both names, but I kept `fd` as the default on the assumption that it was Debian renaming things. If `fdfind` is the popular name, I can make that the default. Or, we can remove the default and use `fdfind` unconditionally. Thoughts?

Reply:
`fd` seems to be the right name to me. `fd-find` being the name of the crate which provides the `fd` CLI.

I think being able to change the name of the CLI is great to have. We ensure the script will work everywhere, whatever the path/name of the `fd` CLI is.